extension/package.json

```diff
@@ -590,6 +595,15 @@
           "description": "%config.pythonPath.description%",
           "type": "string",
           "default": null
+        },
+        "dvc.studioAccessToken": {
```

Comment:
[I] Need to change to use the `SecretStorage` API and not save the token in the open...